do-dont/src/DoDont.ts

```diff
@@ -0,0 +1,144 @@
+import { LitElement, html, svg, css, TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+
+const styles = css`
```

Comment:
I'd move styles to a separate file as we do for all other components in the repository
if you follow my recommendation to use individual files per class, then it will be in `InstructionsCard.styles.js`